tools/coqdep.ml

```diff
@@ -482,9 +107,8 @@ let treat_coqproject f =
   iter_sourced (fun f -> treat_file None f) (all_files project)
 
 let rec parse = function
+  (* Option -c is to be deprecated *)
```

Comment:
```suggestion
  (* TODO deprecate option -c *)
```
easier to grep "TODO" than "is to be"

Reply:
Can't you deprecate in this PR?

Reply:
It is still used by CoqMakefile.in so I'm not sure what the todo here would be, I prefer to do in another PR.

Reply:
Comment updated, thanks.